modules/rostests/apitests/user32_dynamic/load.c

```diff
@@ -18,104 +19,129 @@ START_TEST(load)
     PPEB Peb = NtCurrentPeb();
 
     /* Before init */
+    SetLastError(0xdeadbeef);
     hUser32 = GetModuleHandleW(L"user32");
-    ok(hUser32 == NULL, "hUser32 = %p\n", hUser32);
+    Error = GetLastError();
+
+    if (hUser32 != NULL)
+    {
+        win_skip("user32.dll is already loaded since Vista\n");
+        return;
+    }
+
+    ok(Error == ERROR_MOD_NOT_FOUND, "Error = %lu\n", Error);
     ok(Peb->KernelCallbackTable == NULL, "KernelCallbackTable = %p\n", Peb->KernelCallbackTable);
     ok(Peb->PostProcessInitRoutine == NULL, "PostProcessInitRoutine = %p\n", Peb->PostProcessInitRoutine);
 
-    SetLastError(12345);
+    /* 1st load */
+    SetLastError(0xdeadbeef);
     hUser32 = LoadLibraryW(L"user32");
-    Error = GetLastError();
-    ok(hUser32 != NULL, "LoadLibrary failed\n");
-    ok(Error != 12345, "Error = %lu\n", Error);
```

Comment:
Why remove these tests? Is behavior inconsistent between Windows versions?
(and as a rule, _this_ is the kind of thing that belongs in the PR description and/or commit message. Not just "rewrite", but "rewrite to fix problem xyz")